CONTRIBUTING.md

```diff
@@ -1,52 +1,160 @@
+- [vulnerability](https://github.com/search?q=user%3Agliff-ai+label%3Avulnerability+is%3Aopen) – Security vulnerability report only.
+- [question](https://github.com/search?q=user%3Agliff-ai+label%3Aquestion+is%3Aopen) – Further information is requested and/or required.
+
+**Improvements**  (blue)
+- [enhancement](https://github.com/search?q=user%3Agliff-ai+label%3Aenhancement+is%3Aopen) – New feature or enhancement request.
+- [test](https://github.com/search?q=user%3Agliff-ai+label%3Atest+is%3Aopen) – Anything related to testing or tests.
+- [ui/ux](https://github.com/search?q=user%3Agliff-ai+label%3Aui%2Fux+is%3Aopen) – User interface / user experience of the platform.
+- [deployment](https://github.com/search?q=user%3Agliff-ai+label%3Adeployment+is%3Aopen) – Anything relating to deployment.
+- [documentation](https://github.com/search?q=user%3Agliff-ai+label%3Adocumentation+is%3Aopen) – Improvements or additions to documentation.
+
+**Release**  (yellow)
+- [major](https://github.com/search?q=user%3Agliff-ai+label%3Amajor+is%3Aopen) – Major release.
+- [minor](https://github.com/search?q=user%3Agliff-ai+label%3Aminor+is%3Aopen) – Minor release.
+- [patch](https://github.com/search?q=user%3Agliff-ai+label%3Apatch+is%3Aopen) – Patch release.
 
-If you write new code, we'd appreciate it if you used the same linters and formatters.
-We'd also like new code to come with new tests, where appropriate, and to be run against all existing tests.
+**Inactive**  (black)
+- [wont-fix](https://github.com/search?q=user%3Agliff-ai+label%3Awont-fix+is%3Aopen) – This will not be worked on.
+- [duplicate](https://github.com/search?q=user%3Agliff-ai+label%3Aduplicate+is%3Aopen) – This issue or pull request already exists.
 
-## Submitting changes
+These labels are mostly consistent across gliff.ai repositories^ to make it easier for everyone while working across multiple. If you forget this don't worry, when you open your issue or pull request each label has a description!
 
-Please send a GitHub Pull Request to gliff.ai with a clear list of what you've done (read more about [pull requests](http://help.github.com/pull-requests/)).
-When you send a pull request, we will love you forever if you include tests and documentation.
-We can always use more test coverage.
+**^ NOTE:** the [ROADMAP repository](https://github.com/gliff-ai/roadmap/blob/main/README.md) has its own unique labelling system.
 
-Always write a clear log message for your commits.
-One-line messages are fine for small changes, but bigger changes should look like this:
+## Contact
 
-    $ git commit -m "feat: Add new feature
-    >
-    > A paragraph describing what changed and its impact."
+[{{back to navigation}}](#table-of-contents)
 
-Thanks,
-The gliff.ai team
+Need some help? 🤔 Have a question? 🧠 \
+Reach out to the gliff.ai team at [contact@gliff.ai](mailto:contact@gliff.ai?subject=[GitHub]) or on GitHub.
```

Comment:
Maybe this should be a #techteam shared "community@gliff.ai" and, again, be autoforwarded to Slack?

Reply:
I think that's a better idea! I'll change emails across documentation now. Do we want the autoforwarded messages to be in a separated "github" channel for slack? As we did with other alerts etc?

Reply:
Yes, probably. Let's touch base on emails/slack integration stuff tomorrow.